### ISSUE TITLE
Update build on Windows

### ISSUE THREAD

**Issue by skalarproduktraum** [AdaptiveParticles/LibAPR#138]
With Intel's C++ compiler now being free on Windows as well (see https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/dpc-compiler.html), there's a better path for compiling libAPR on Windows.

This however hinges on two issues:

* [ ] better CMake support for Intel C++ Compiler 2021. New version of CMake is due in a few days, let's see if that changes
* [ ] pyLibAPR is using cmake-setuptools, which thinks `make` is the proper way to run a build with cmake in general. A PR is open there (https://github.com/raydouglass/cmake_setuptools/pull/1) since some time with no reaction from the developer. We might need a better setup toolchain there. 

Here's also what I have run so far (on top of miniconda3 on Windows 10), with some comments:

```
pip install virtualenv
python -m virtualenv libapr-env
virtualenv libapr-env
pip install -r requirements.txt
# numpy bailed, installed manually, works then
python setup.py install
# bailed on HDF5
conda install hdf5 # for hdf5 thingy
# bailed on libtiff
conda install libtiff
```

**Comment by cheesema** [comment]
#140 some success here.

**Comment by cheesema** [comment]
Compilation issues are solved in #142 however, I have yet to then look at the follow through issues for python.

**Comment by cheesema** [comment]
This is now resolved, it hasn't been pushed through yet to master on PyLibAPR, but is working across Linux, Windows, and OSX, with OpenMP support pre-release can be found here: https://test.pypi.org/project/PyLibAPR-test/#files
